packages/trader/src/Modules/Trading/Components/Form/TradeParams/Turbos/barrier-selector.jsx

```diff
@@ -140,6 +114,7 @@ const BarrierSelector = ({ barrier_1, onChange, setHoveredBarrier, turbos_barrie
                     </Text>
                     <Text size='xs' className='trade-container__barriers-value' data-testid='current_barrier'>
                         {barrier_1}
+                        <Icon icon='IcChevronLeft' className='trade-container__barriers-value-arrow' />
```

Comment:
```suggestion
                        <Icon icon='IcChevronLeft' className='trade-container__barriers-value--arrow-right' />
```
@kate-deriv at least classname will tell to us that this arrow is actually turned to the right )

Reply:
Done, thank you